packages/core/content-type-builder/admin/src/components/FormModal/FormModal.tsx

```diff
@@ -1008,7 +1008,7 @@ export const FormModal = () => {
           />
         )}
         {!isPickingAttribute && (
-          <form onSubmit={handleSubmit}>
+          <form onSubmit={handleSubmit} style={{ overflow: 'auto' }}>
```

Comment:
Instead of using an inline style can I ask you to create a styled component on top of the FormModal one
```suggestion
          import { styled } from 'styled-components';
```
```suggestion
         const FormComponent = styled.form`
  overflow: auto;
`;
```
```suggestion
         <FormComponent onSubmit={handleSubmit}>...</FormComponent>
`;
```

Reply:
@simotae14 thanks for suggestions I updated code as per suggestions